src/transformers/onnx/config.py

```diff
@@ -13,15 +13,31 @@
 # limitations under the License.
 import copy
 import dataclasses
+import warnings
 from abc import ABC, abstractmethod
 from collections import OrderedDict
-from typing import Any, Callable, Dict, Iterable, List, Mapping, Optional, Tuple
+from typing import Any, Callable, Dict, Iterable, List, Mapping, Optional, Tuple, Union
 
-from transformers import PretrainedConfig, PreTrainedTokenizer, TensorType, is_torch_available
+import numpy as np
+from packaging import version
 
+from transformers import (
+    PretrainedConfig,
+    PreTrainedTokenizer,
+    PreTrainedTokenizerFast,
+    TensorType,
+    is_torch_available,
+    is_vision_available,
+)
```

Comment:
We shouldn't have absolute imports mixed with relative imports, so everything should be relative imports.
Also `PreTrainedTokenizerFast` should not be imported outside of a check for `is_tokenizers_available` but I don't think you really need it.

Reply:
Thanks for the advice. Here I was just following what already existed, but I'll take care of the relative imports in this PR too :)